src/plugins/htmlAudioPlayer/plugin.js

```diff
@@ -112,9 +112,14 @@ class HtmlAudioPlayer {
             let val = options.url;
             console.debug('playing url: ' + val);
             import('../../scripts/settings/userSettings').then((userSettings) => {
-                if (userSettings.enableAudioNormalization() && options.item.LUFS != null) {
+                if (userSettings.selectAudioNormalization() == 'TrackGain' && options.item.LUFS != null) {
                     const dbGain = -18 - options.item.LUFS;
                     self.gainNode.gain.value = Math.pow(10, (dbGain / 20));
+                    console.debug('Using track gain');
```

Comment:
Are these debug statements still useful? If so I would add the class name as a prefix:
```suggestion
                    console.debug('[HtmlAudioPlayer] Using track gain');
```

Reply:
I have added/updated these will the latest push